### PR TITLE
Updated third parties with CVEs and removed unused packages.config files

### PR DESCRIPTION
**Pull request by SimbaGithub**
Third parties are updated to account for the CVEs. And the unused "packages.config" files are removed since it caused some CVEs to show up even though the packages listed in those files are not installed or used by the project

### CODE REVIEW

File: Snowflake.Data/Snowflake.Data.csproj [hunk 1]
```diff
@@ -19,7 +19,8 @@
   <ItemGroup>
     <PackageReference Include="AngleSharp" Version="0.12.1" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="log4net" Version="2.0.8" />
+    <PackageReference Include="log4net" Version="2.0.11" />
+    <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
```
Comment (sfc-gh-hyu) [inline Snowflake.Data/Snowflake.Data.csproj:23]: Just curious, why this is package reference is needed?

File: Snowflake.Data.Tests/Snowflake.Data.Tests.csproj [hunk 2]
```diff
@@ -29,6 +30,12 @@
     <Reference Include="System.Web" />
   </ItemGroup>
   
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+    <PackageReference Include="Microsoft.PowerShell.Commands.Utility">
```
Comment (sfc-gh-hyu) [inline Snowflake.Data.Tests/Snowflake.Data.Tests.csproj:34]: What is powershell commands utitlity used for?

File: Snowflake.Data/Snowflake.Data.csproj (hunk 1 above)
Comment (SimbaGithub) [reply Snowflake.Data/Snowflake.Data.csproj:23]: Hi, System.Text.RegularExpressions is a dependency of log4net
And it has a vulnerability which version 4.3.1 fixes: https://www.whitesourcesoftware.com/vulnerability-database/CVE-2019-0820

File: Snowflake.Data.Tests/Snowflake.Data.Tests.csproj (hunk 2 above)
Comment (SimbaGithub) [reply Snowflake.Data.Tests/Snowflake.Data.Tests.csproj:34]: Powershell is not actually used by the driver. But .NET Core 2.0 has a vulnerability which upgrading Powershell to 6.1.0 is part of the fix: https://www.whitesourcesoftware.com/vulnerability-database/CVE-2018-8292

**Review by sfc-gh-hyu — APPROVED** [review]
Thanks for the explanation. LGTM.

**Pull request merged**